dev/fattest.simplicity/src/componenttest/topology/utils/HttpRequest.java

```diff
@@ -274,6 +282,10 @@ public int getResponseCode() {
         return responseCode;
     }
 
+    public Map<String, String> getResponseHeaders() {
+        return responseHeaders;
+    }
```

Comment:
This causes a compile failure in `com.ibm.ws.ui_rest_fat` because they extend `HttpRequest` and add a method `public Map<String,List<String>> getResponseHeaders()`